webpack.config.js

```diff
@@ -1,33 +1,32 @@
-const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
   mode: 'development',
   entry: {
     index: './src/index.js',
-    print: './src/print.js',
+    print: './src/print.js'
   },
   devtool: 'inline-source-map',
   devServer: {
-    static: './dist',
+    static: './dist'
   },
   module: {
     rules: [
       {
         test: /\.css$/i,
-        use: ['style-loader', 'css-loader'],
-      },
-    ],
+        use: ['style-loader', 'css-loader']
+      }
+    ]
   },
   plugins: [
     new HtmlWebpackPlugin({
-      title: 'Development',
-    }),
+      title: 'Development'
+    })
   ],
   output: {
     filename: '[name].bundle.js',
     path: path.resolve(__dirname, 'dist'),
     clean: true,
-    publicPath: '/',
-  },
+    publicPath: '/'
+  }
 };
```

Comment:
### I'll leave comments here as you did not change most of your files. 

- [ ] **Required:**

![image](https://user-images.githubusercontent.com/27709832/171420551-4c13c96d-01d1-4611-80a3-31edc8274065.png)
- Update the name of your project. The **Project Name** is a placeholder.
- Your **MIT** license is empty. You can populate it with [the template here.](https://github.com/microverseinc/readme-template)